src/market/scan/scanned-offer.ts

```diff
@@ -145,14 +145,15 @@ export class ScannedOffer {
    * ```
    */
   get paymentPlatformAddresses(): PaymentPlatformAddressSet {
-    const platformProps = Object.entries(this.model.properties).filter(([key]) =>
-      key.startsWith("golem.com.payment.platform."),
-    );
-
-    const platformAddress = platformProps.map(([key, address]) => [
-      key.replace("golem.com.payment.platform.", "").replace(".address", ""),
-      address,
-    ]);
+    const propRegex = /golem.com.payment.platform.([a-z0-9-]+).address/;
+    const platformProps = Object.entries(this.model.properties).filter(([key]) => key.match(propRegex));
```

Comment:
Is this first filter needed anymore? You're filtering again based on the same condition below

Reply:
As per our call: It's not needed, I will drop it.